shenyu-kubernetes-controller/src/main/java/org/apache/shenyu/k8s/reconciler/IngressReconciler.java

```diff
@@ -439,4 +433,24 @@ private void addNewIngressConfigToShenyu(final V1Ingress v1Ingress, final CoreV1
             }
         }
     }
+
+    private String getPluginName(final V1Ingress ingress) {
+        String pluginName;
+        String pluginDubboEnabled = ingress.getMetadata().getAnnotations().get(IngressConstants.PLUGIN_DUBBO_ENABLED);
+        String pluginMotanEnabled = ingress.getMetadata().getAnnotations().get(IngressConstants.PLUGIN_MOTAN_ENABLED);
+        String pluginSpringCloudEnabled = ingress.getMetadata().getAnnotations().get(IngressConstants.PLUGIN_SPRING_CLOUD_ENABLED);
+        String pluginWebSocketEnabled = ingress.getMetadata().getAnnotations().get(IngressConstants.PLUGIN_WEB_SOCKET_ENABLED);
+        if ("true".equals(pluginDubboEnabled)) {
```

Comment:
Boolean.TRUE

Reply:
ping @runqi-zhao

Reply:
Ok I'm going to modify it using BOLEAN.TRUE.toString